config/defaults.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'items_per_page' => 10,
```

Comment:
This will not work, because the value from this config is used by the laravel-helpers package. I think it’s better to just delete this config and correct the fixtures with the test results. I did it.